service/src/test/java/greencity/service/UBSManagementServiceImplTest.java

```diff
@@ -561,6 +552,38 @@ void updateOrderDetailStatus() {
         verify(eventService, times(1))
             .save("Статус Замовлення - Скасовано" + "  " + order.getCancellationComment(),
                 user.getRecipientName() + "  " + user.getRecipientSurname(), order);
+
+    }
+
+    @Test
+    void updateOrderDetailStatusSecond() {
+        User user = ModelUtils.getTestUser();
+        user.setRecipientName("Петро");
+        user.setRecipientSurname("Петренко");
+        when(userRepository.findUserByUuid(user.getUuid())).thenReturn(Optional.of(user));
+        Order order = user.getOrders().get(0);
+        order.setOrderDate((LocalDateTime.of(2021, 5, 15, 10, 20, 5)));
```

Comment:
di you really need this specific date? use just LocalDateTime.now() or something